Packages/Account/Sources/Account/AccountDetailHeaderView.swift

```diff
@@ -193,6 +199,23 @@ struct AccountDetailHeaderView: View {
       .padding(.top, 6)
     }
   }
+  
+  @ViewBuilder
+  private func makeNoteView(_ note: String) -> some View {
+    VStack(alignment: .leading, spacing: 4) {
+      Text("account.relation.note.label")
+        .foregroundColor(.gray)
+      Text(note)
+        .frame(maxWidth: .infinity)
```

Comment:
Could you add alignment: .leading to fix the one line issue? Not by a computer to confirm if it works.

Reply:
The `Text` has no `.alignment`, and on the `VStack` containing the text it is already set.